spectool/jdspec.ts

```diff
@@ -571,6 +572,14 @@ export function parseServiceSpecificationMarkdownToJSON(
         if (words[0] === "internal") {
             internal = true
             words.shift()
+        } 
+        
+        let volatile: boolean = undefined
```

Comment:
maybe automatically mark reading registers as volatile

Reply:
that is done automatically by the system via inheritance.

Reply:
my bad... need to do that.

Reply:
Is it OK to have a service that has a reading register but doesn't extend _sensor?

Reply:
yes, at least jacdac-docs ignore the sensor base class info since all packets are flattened when processing the specs.